airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -159,12 +159,84 @@ def get_start_timestamp(self, stream_state) -> int:
 
         return start_point
 
+class IncrementalStripeStreamWithUpdates(IncrementalStripeStream):
+    """
+        This is a base class for incremental streams that support updates using the events API
+        In first sync it will not get any updates as the records are already updated
```

Comment:
As we are adding new fields like `event_created`, `{stream_name}_{primary_key}` or `rs_id` to the records, these should exist even for first sync records for these resources. Please check this comment in design doc.
https://www.notion.so/Stripe-updated-records-d1df062071b848cb87966c1b2d56f8a1?d=42fadec1d8444b09be62ed6374395cdc#7895be5b672d466696d973c564c16698

Reply:
Yeah this is handled by [parse_response](https://github.com/rudderlabs/airbyte/pull/30/files#diff-bc939ec7f5dc075809864e54aca658012af002c91181af90b6b058de17bc4205R217)